chord_metadata_service/mohpackets/data/data_converter.py

```diff
@@ -163,7 +163,8 @@ def main():
         print("Invalid option. Please try again.")
         return
 
-    set_foreign_keys(path)
+    program_id = str(input("Enter the dataset name (default: SYNTHETIC-): ")) or "SYNTHETIC-"
```

Comment:
Thinking about this @mshadbolt -- this allows people to change the prefix of the dataset (e.g. from `SYNTHETIC-2` to `TEST-2`), but does not allow people to replace the range afterwards -- is that good?

Reply:
I think it is ok, given it isn't really that clear to a user how many datasets a user is converting with the script, I think expecting them to come up with multiple names rather than having a sequential number at the end is too complicated. A customizable prefix with sequential numbers and should be suitable for what we need here. Thanks for checking in!